CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [0.22.727](https://github.com/brave/browser-laptop/releases/tag/v0.22.727dev)
+
+ - Added Captcha verification when accepting Payment grants. ([#14188](https://github.com/brave/browser-laptop/issues/14188))
+ - Added "Clear Payment Data / Clear Payment History" under security preferences and "Delete Wallet" under advanced payments preferences. ([#8537](https://github.com/brave/browser-laptop/issues/8537))
+ - Optimized contributions by adding support for batch voting. ([#14190](https://github.com/brave/browser-laptop/issues/14190))
+ - Fixed Payments displaying message stating that 30 minutes of total usage hasn't been logged even though table contains more than 30 minutes of usage. ([#14310](https://github.com/brave/browser-laptop/issues/14310))
```

Comment:
might be better captured as `Fixed Payments displaying message incorrectly stating that 30 minutes of total usage hasn't been logged (in some situations)`

Reply:
we could also include that purging is not reflected in the table (which was not the case before that, synopsis would be purged, but table would not be so you would be stuck in pushback hell)